### DISCUSSION TITLE
umi3中这个插件react-dev-inspector的功能在umi4有替代方案吗？

### DISCUSSION THREAD

**Discussion by coolMan2015** [umijs/umi#11989] [Q&A]
react-dev-inspector这个插件能通过界面快速定位到对应的源文件，非常方便，不知在umi4咋去掉了，还没有办法集成，请问有什么替代方案？

**Comment by fz6m** [comment]
看下 react-dev-inspector 官方的 [example](https://github.com/zthxxx/react-dev-inspector/tree/dev/examples/umi4) 吧，应该是可用的。
